### ISSUE TITLE
text box configurations

### ISSUE THREAD

**Issue by princerosa** [antero111/plugin-presets#21]
Hello, I was using plugin pre-set plugin to change between different custom menu swap configurations , and it worked great but now after plugin pre-sets has been updated it doesn't work the way it did before. Now the way I understand it, when you pre-set configuration only allows for you to toggle the checkboxes on a preset on/off, but not edit the text in a box - ie: which menu swaps you want. I hope this makes sense? Does anyone know a fix? Manually editing the text in the custom swaps box as i need is a drag.

Thanks!

**Comment by princerosa** [comment]
Sure, so I use the plugin below to swap between different configurations when tobbing, to make my preferred nylo the default left click option in a stack. Before the plugin presets was updated I was able to save different configurations (edits to the text box of this plugin) to different presets. Now as far as i can tell I can't do that with the presets plugin.

![image](https://user-images.githubusercontent.com/110763629/183776347-34f80298-9053-4b90-9b75-e6d0467c9beb.png)

I'm trying to be able to make changes to the text boxes within other plugins with the presets plugin.

**Comment by antero111** [comment]
> I'm trying to be able to make changes to the text boxes within other plugins with the presets plugin.

I'm, sorry but I dont quite understand what are you telling here but I just tested and adding custom hides (Custom Menu Swaps) configurations works fine.
 
![kuva](https://user-images.githubusercontent.com/60603897/183973774-1f179c6c-1384-4f33-9ebe-05dc20af0a98.png)

You can edit plugin settings one by one by clicking the plugin name. 

Afaik in your case you wanted different presets for tob roles. To do this, make those role presets and then save different configurations for the menu swaps in those presets.

In short: edit role x preset, open RL configurations tab and edit your custom swaps, update custom swaps configurations (make sure the custom swaps/custom hides checkbox is checked so its included in the preset) and then repeat with other role presets.

From the readme: _The checkboxes visualize whether or not the preset has stored any configurations to a certain plugin. If the checkbox is checked, that preset has stored some configurations and loading it will enable those configurations for you._
